packages/ui/src/components/Layout.tsx

```diff
@@ -0,0 +1,47 @@
+import StorageOverview from './StorageOverview'
+import PlayerDisplay from './PlayerDisplay'
+import NavBar from './NavBar'
+
+interface Props {
+  children: React.ReactNode
+}
+
+function Layout({ children }: Props) {
+  return (
+    <>
+      <NavBar
+        tabs={[
+          { name: 'Cockpit', href: '/play/cockpit' },
+          { name: 'Cockpit', href: '/play/cockpit' },
+        ]}
+        playerName={'Team1'}
+        playerLevel={2}
+      />
+      <div className="flex w-full justify-between gap-x-4 p-4">
+        {children}
+        <div id="sidebar" className="flex w-60 flex-col">
+          <PlayerDisplay
+            name={'playerName'}
+            imgPathAvatar={'/avatars/avatar_placeholder.png'}
+            color={'Red'}
+            achievements={[]}
+            imgPathLocation={'/locations/ZH.svg'}
+            location={'ZH'}
```

Comment:
```suggestion
            location="ZH"
```

provide strings like this, the curly brackets are just needed for "dynamic" content or numbers, makes it a bit quicker to distinguish.

same with booleans, which can be passed like <Toggle active /> instead of <Toggle active={true} />, and if the default is set to false (sensible for booleans), if they are not there, they are false

Reply:
Alright nice, thanks for the info!